packages/assistant-stream/src/core/effects/ToolExecutionStream.ts

```
@@ -0,0 +1,118 @@
+              const argsText = toolCallArgsText[toolCallId];
+              if (!argsText)
+                throw new Error("Unexpected tool call without args");
+
+              const executeTool = async () => {
+                let args;
+                try {
+                  args = sjson.parse(argsText);
+                } catch (e) {
+                  throw (
+                    "Function parameter parsing failed. " +
+                    JSON.stringify((e as Error).message)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)